core/web/resolver/spec.go

```diff
@@ -125,6 +127,10 @@ func (r *SpecResolver) ToWorkflowSpec() (*WorkflowSpecResolver, bool) {
 	return &WorkflowSpecResolver{spec: *r.j.WorkflowSpec}, true
 }
 
+func (r *SpecResolver) ToStandardCapabilitiesSpec() (*StandardCapabilitiesSpecResolver, bool) {
+	return nil, false
```

Comment:
These are not turned on in the pipeline for 2.12, and there shouldn't be any  StandardCapabilitiesSpecs, so it shouldn't be any issue, other than having this to support the schema for the existing UI changes.